js-project-guidelines.md

```diff
@@ -191,53 +226,62 @@ If you install the module through npm and require it, you receive the ES5 versio
 var API = require(‘ipfs-api’)
 ```
 
-If you use module bundler that understands ES2015 like webpack@2 or rollup you can use this syntax to get the original ES2015 source.
+If you use a module bundler that understands ES2015 like webpack@2 or rollup you can use this syntax to get the original ES2015 source.
 
 ```js
 const API = require(‘ipfs-api/src’)
 ```
 
-### FAQ
+## FAQ
+
 
 #### Why are you not using XYZ?
 
-There are two possibilities, either it didn’t work out for us, or we don’t know about it. If you think we might have missed it please tell us, but please believe us if we say we tried and it didn’t work for us.
+There are two possibilities: either it didn’t work out for us, or we don’t know about it. If you think we might have missed it please tell us, but please believe us if we say we tried and it didn’t work for us.
 
-#### Ugh, gulp why not use simple npm scripts?
+#### Why not use simple npm scripts instead of gulp?
 
-Gulp is not a hard dependency, it’s just a simple way to structure our tasks at the moment. Usually projects only depend on the dignified binaries completely hiding the fact that we are using gulp under the hood. So we are free if we want to switch it out without any issues. We all enjoy npm scripts, and are using them to call the dignified binaries, but there is no nice way of sharing them yet.
+Gulp is not a hard dependency. It’s just a simple way to structure our tasks at the moment. Usually projects only depend on the aegir binaries completely hiding the fact that we are using gulp under the hood. So we are free if we want to switch it out without any issues. We all enjoy npm scripts, and are using them to call the aegir binaries, but there is no nice way of sharing them yet.
 
 #### Where are all the semicolons?
 
-Our linting rules are compatible with [standard](https://github.com/feross/standard) which has many examples on documentation on this. Please go there and read it.
+Our linting rules are compatible with [standard](https://github.com/feross/standard), which has many examples on documentation on this. Please go there and read it if you're still curious.
 
 #### Why are you bothering with ES2015 and all this build setup?
 
-We want to see the web move forward, and some of us enjoy writing their JavaScript with things like const and arrow functions.
+We want to see the web move forward, and some of us enjoy writing their JavaScript with things like `const` and arrow functions.
+
+#### Do I have to use ES2015 and Babel and aegir.js in my project?
+
+No.
+
+#### Do I have to bundle everything with webpack?
+
+No. But other people might ask you to at some point, so, maybe it's better to be prepared?
 
-#### Why are doing this?
+#### Why are you doing this?
```

Comment:
"This tooling was borne of hard learning done from years in the trenches. Its goal is to minimize process road bumps and provide a unified low-friction workflow for contributors."